tests/src/FunctionalJavascript/ArticleCreationTest.php

```diff
@@ -119,6 +119,27 @@ public function testCreateArticle() {
       ->elementsCount('xpath', '//div[contains(@class, "field--name-field-paragraphs")]/div[contains(@class, "field__item")][9]//span[contains(@data-pin-id, "478085316687452268")]', 2);
   }
 
+  /**
+   * Test Creation of Article without content moderation.
+   */
+  public function testCreateArticleWithNoModeration() {
+    // Delete all the articles so we can disable content moderation.
+    foreach (\Drupal::entityTypeManager()->getStorage('node')->loadByProperties(['type' => 'article']) as $node) {
```

Comment:
Interesting. Locally I had to remove all the moderation states in order to disable the content_moderation module. The content could stay for me.

Reply:
Yeah that would work... I just did it the clean way

Reply:
Ok funny. The uninstall form recommends deleting the moderation states, but removing the content also works.
Anyway...